src/pages/profile/profile.css

```diff
@@ -46,8 +45,35 @@
   color: var(--primary-color);
   border-bottom: var(--primary-border);
 }
-.signout-btn {
-  align-self: flex-start;
+.quiz-stats {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border: 1px solid var(--secondary-color);
+  /* background-color: var(--secondary-color);
+  color: var(--primary-color); */
+  border-radius: 5px;
+  /* justify-content: space-between; */
```

Comment:
You can remove the comments which are not required

Reply:
done